.github/workflows/deploy-zeit-staging.yml

```diff
@@ -147,35 +139,83 @@ jobs:
           name: videos
           path: cypress/videos/
 
-      # We need to find the PR id. Will be used later to comment on that PR.
```

Comment:
Because now, we trigger the actions on a "pull_request" and not a "push". By this way, it's so much easier to find the PR id, we just have to use the variable `${{ github.event.number }}`

Reply:
Great to know!